### DISCUSSION TITLE
how can I get the message info with id?

### DISCUSSION THREAD

**Discussion by nekooee** [def-studio/telegraph#361] [Q&A]
I have a message-id and chat-id. how can I get message info??

**Comment by fabio-ivona** [comment]
Hi, how did you obtain the message ID?

**Reply by nekooee** [reply]
> Hi, how did you obtain the message ID?

from the post link that a user sends with a request to the server.
for example:
https://t.me/front_end_first/4450
"front_end_first" is the channel name and "4450" is the message ID. if the channel has been private, the post link included the channel ID instead of the channel name. Now I need to get the message info and then find the file ID.

**Reply by fabio-ivona** [reply]
Telegram APIs don't support any way to retrieve a message by id
